src/resources/springboot/application.properties

```diff
@@ -0,0 +1,6 @@
+# expose the actuator endpoints
+#  http://localhost:8080/actuator/beans
+#  http://localhost:8080/actuator/info
+management.endpoints.web.exposure.include=*
+spring.profiles.active=dev
+spring.output.ansi.enabled=ALWAYS
```

Comment:
Color output will most likely be moved to a development profile/properties file so it's not used in production